tests/data/hpd_matrices_data.py

```diff
@@ -0,0 +1,480 @@
+import math
+import random
+
+import geomstats.backend as gs
+from geomstats.geometry.hpd_matrices import (
+    HPDAffineMetric,
+    HPDBuresWassersteinMetric,
+    HPDEuclideanMetric,
+    HPDLogEuclideanMetric,
+    HPDMatrices,
+)
+from tests.data_generation import _ComplexRiemannianMetricTestData, _OpenSetTestData
+
+SQRT_2 = math.sqrt(2.0)
+LN_2 = math.log(2.0)
+EXP_1 = math.exp(1.0)
+EXP_2 = math.exp(2.0)
+SINH_1 = math.sinh(1.0)
+
+
+class HPDMatricesTestData(_OpenSetTestData):
```

Comment:
Let's generate data that is complex from scratch, instead of casting in tests (so after that, remove casting from test file).

Reply:
Done.